common/common/src/main/java/io/helidon/common/GenericType.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The `All rights reserved.` should be removed. Not sure why the copyright plugin ignores this

Reply:
I've only updated the year in these copyright changes, can we do that change in a separate PR ?

Reply:
Marking this as resolved since I think we should solve this issue for all of Helidon not just for this PR.